multihost_test/Bugzillas/test_bz_automation.py

```diff
@@ -355,3 +365,202 @@ def test_bz_951743(self, multihost):
         execute_cmd(multihost, "userdel -rf bz951743")
+
+    @pytest.mark.tier1
+    def test_bz690829_1(self, multihost):
+        """
+        :title: Checks if useradd is able to copy files
+            from /etc/skel also if /home not mounted with acl option.
+        :id: 5d372fe0-4c88-11ee-9152-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=690829
+        :steps:
+          1. we are expecting /home exists
+          2. create files/dirs under skel
+          3. create new user
+          4. check if setfacl doesn't work
+          5. check if all files copied and correctly
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+          5. Should succeed
+        """
+        # we are expecting /home exists
+        execute_cmd(multihost, "dd if=/dev/zero of=homedisk bs=1M count=200")
+        execute_cmd(multihost, "losetup /dev/loop0 homedisk")
+        execute_cmd(multihost, "mkfs.ext3 /dev/loop0")
+        execute_cmd(multihost, "mount -o noacl /dev/loop0 /home")
+        execute_cmd(multihost, "restorecon -RvvF /home")
+        # create files/dirs under skel
+        execute_cmd(multihost, "mkdir -p /etc/skel/a/b/c/d/e/f")
+        execute_cmd(multihost, "echo test > /etc/skel/a/b/c/d/e/f/tfile")
+        execute_cmd(multihost, "dd if=/dev/zero of=/etc/skel/a/b/c/bigfile bs=1M count=100")
+        execute_cmd(multihost, "echo selinux > /etc/skel/a/suppa")
+        execute_cmd(multihost, "chcon -t etc_t /etc/skel/a/suppa")
+        execute_cmd(multihost, "echo selinux > /etc/skel/suppa")
+        execute_cmd(multihost, "chmod 444 /etc/skel/suppa")
+        execute_cmd(multihost, "chcon -t etc_t /etc/skel/suppa")
+        # create new user
+        execute_cmd(multihost, "useradd test_anuj")
+        # check if setfacl doesn't work
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "setfacl -m u:test_anuj:rwx /home/test_anuj/suppa")
+        # check if all files copied and correctly
+        assert execute_cmd(multihost, "test -f /home/test_anuj/a/b/c/d/e/f/tfile").returncode == 0
+        assert execute_cmd(multihost, "du -hs /home/test_anuj/a/b/c/bigfile | egrep 10?M").returncode == 0
+        assert execute_cmd(multihost, "cat /home/test_anuj/a/b/c/d/e/f/tfile | grep test").returncode == 0
+        assert execute_cmd(multihost, "cat /home/test_anuj/suppa | grep selinux").returncode == 0
+        assert execute_cmd(multihost, "ls -Z /home/test_anuj/suppa | grep home_t").returncode == 0
+        assert execute_cmd(multihost, "ls -Z /home/test_anuj/a/suppa | grep home_t").returncode == 0
+        # Clean_up
+        clean_up(multihost)
+
+    @pytest.mark.tier1
+    def test_bz690829_2(self, multihost):
+        """
+        :title: Checks if useradd is able to copy files
+            from /etc/skel also if /home not mounted with acl option.
```

Comment:
IIUC, this test case is mounted with acl option, but you state otherwise in the `title`

Reply:
This comment also applies to test case 4